### DISCUSSION TITLE
Failed to reach ring server?

### DISCUSSION THREAD

**Discussion by eleeptg** [dgreif/ring#983] [General]
Recently I’ve noticed this in thee error logs..

Failed to reach Ring server at https://api.ring.com/clients_api/ring_devices.  Timeout awaiting 'request' for 20000ms.  Trying again in 5 seconds...

Any ideas? Ring app works fine and Speedtest is 800mbps up/down so i dont think its a bandwidth issue on my home network.

**Comment by dgreif** [comment]
This usually just means there was a network blip that prevented the request from going through.  Not a big deal if everything is working for you now.

**Comment by eleeptg** [comment]
Thanks.. strange though.. everything appears to be working. Just the error on the console every 2 minutes..

**Reply by r8ders2k** [reply]
FYI, I just updated to the latest Homebridge update and saw that I needed to refresh the Ring token. I followed the refreshToken instructions  and it's back, up 'n running. Thanks.
